src/components/CreateCourseRunPage/index.jsx

```diff
@@ -116,6 +131,8 @@ class CreateCourseRunPage extends React.Component {
     const showSpinner = !startedFetching || courseInfo.isFetching || courseRunOptions.isFetching;
     const showForm = startedFetching && !courseInfo.isFetching && !courseRunOptions.isFetching &&
       !courseInReview;
+    const sortedRunLabels = course_runs && this.parseCourseRunLabels(course_runs);
+    const defaultRun = sortedRunLabels && sortedRunLabels[0].value;
```

Comment:
Since this won't have a default value in the form, you could update this to
`const defaultRun = (sortedRunLabels && sortedRunLabels[0].value) || '';`